packages/transport/test/car.spec.js

```diff
@@ -121,6 +122,8 @@ test('accepts Content-Type as well', async () => {
     expiration,
   })
 
+  CAR.encode([delegation])
```

Comment:
is this just asserting that `encode` doesn't throw? or does the call to `encode` have an important side effect?

Reply:
No just a mistake, I'll remove it